### ISSUE TITLE
Height variable on mobile

### ISSUE THREAD

**Issue by Satif** [gilbitron/carouFredSel#35]
I have a slider with 2 slides. Options: 
{
responsive:true,
height: 'variable'
}

When scroll bottom (on mobile), browser address bar pannel will hide, slider update height and jump to top. And when scroll to top, browser address bar pannel will show and slider update sizes and jump to top too. How fix it?

**Comment by bialabs** [comment]
same here, found a solution?

**Comment by Satif** [comment]
no. I call callback 'updateSizes' some times and carousel calculate mobile browser url bar height) only when scroll top, and only when scroll bottom after this bar hide

**Comment by Satif** [comment]
This issue still open! Can anyone have some solution?

**Comment by Satif** [comment]
Ok. Here is a solution
$(window).scroll(function () {
    $("body").attr("height",$(document).scrollTop());
});

$(window).on("resize", function(e){
    e.preventDefault();
    $(document).scrollTop($("body").attr("height"))
});
